system-addon/content-src/lib/init-store.js

```diff
@@ -58,7 +58,17 @@ module.exports = function initStore(reducers, initialState) {
   );
 
   if (global.addMessageListener) {
+    let didInitialize = false;
```

Comment:
I used this instead of `App.initialized` to decouple this from the specifics of the reducer's logic, in case it ever gets changed.